### PR TITLE
Remove warnings

### PR DESCRIPTION
**Pull request by ThorbenBender**
# Description

I removed all the warnings and errors that appeared in the console. 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, if relevant
- [ ] I have run the app using my feature and ensured that no functionality is broken
- [ ] My changes generate no new warnings
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ function App(props) {
   // ^Needed to stop the screen from scrolling when the modal is visible!
 
   return (
-    <div className="App" style={{height: "100%", "overflow-y": isDisplaying ? "hidden" : "visible" }}>
+    <div className="App" style={{height: "100%", "overflowY": isDisplaying ? "hidden" : "visible" }}>
```
Comment (RodPaDev) [inline src/App.js:25]: is this overflowY a prop?

**Review by RodPaDev — APPROVED** [review]
Looks good

File: src/components/createRecipe/recipeForm/Step1.jsx [hunk 2]
```diff
@@ -14,22 +14,20 @@ import {
...
+//     display: 'flex',
+//     flexWrap: 'wrap',
+//   },
+//   textField: {
+//     marginLeft: theme.spacing(1),
+//     marginRight: theme.spacing(1),
+//     color: 'white'
+//     // font-size: 18px
+//   },
+// }));
```
Comment (carnundotcom) [inline src/components/createRecipe/recipeForm/Step1.jsx:30]: What's up with this one? Is it a Material UI thing?

File: src/components/createRecipe/recipeForm/Step2.jsx [hunk 3]
```diff
@@ -4,24 +4,15 @@ import * as dispatchers from "../../../actions/actionCreators";
...
 import {
-  Section1,
   NavigationSection1,
   Addtitle,
-  Section2,
   Section3,
-  Title,
-  Section2b,
   ExportImg
 } from "./FormStyled.styles";
```
Comment (carnundotcom) [inline src/components/createRecipe/recipeForm/Step2.jsx:13]: Is nothing in this file actually used? :joy:

File: src/components/createRecipe/recipeForm/Step3.jsx [hunk 4]
```diff
@@ -7,21 +7,15 @@ import CheckIcon from "@material-ui/icons/Check";
...
   Addtitle,
   Section2b,
   IngredientsDiv,
-  IngredientsMainDiv,
   IngredientsDiv1,
   IngredientsWrapper,
-  Title2,
   Section3,
-  Section1,
   AddItem
```
Comment (carnundotcom) [inline src/components/createRecipe/recipeForm/Step3.jsx:19]: Here too, ha.

Comment (carnundotcom) [reply src/components/createRecipe/recipeForm/Step3.jsx:19]: And in other Step files...

File: src/reducers/reducers.js [hunk 5]
```diff
@@ -312,84 +312,85 @@ const initialProfileState = {
...
+//   recipe_count: 2, // A
+//   recipes_forked_count: 1, // B
+//   // And the following are meta:
+//   isFetchingProfile: false,
+//   isFetchingUserRecipes: false,
+//   isFetchingUserLikes: false,
+//   isFetchingForkedRecipesCount: false,
+//   error: "",
+//   message: ""
+// };
```
Comment (carnundotcom) [inline src/reducers/reducers.js:394]: My bad!

**Review by carnundotcom — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (ThorbenBender) [reply src/App.js:25]: was one of the errors I was getting. doesn't even make sense that the key has double quotes

File: src/components/createRecipe/recipeForm/Step1.jsx (hunk 2 above)
Comment (ThorbenBender) [reply src/components/createRecipe/recipeForm/Step1.jsx:30]: just normal styling but never used

**Pull request merged**